Docs/checklist-for-new-user-facing-features.md

```diff
@@ -0,0 +1,47 @@
+## New Feature Development Checklist
+
+So you've added a new feature. Users will see it!
+
+You've landed the code, it's indented properly, it looks lovely! What's Next!?
+
+###  Has your feature got tests yet?
+
+  * perhaps existing tests will cover it, may be adding another unit test will be enough.
+  * If you're adding new UI:
+    - ensure widgets have `accessbilityIdentifiers`. 
+    - Add it to FxScreenGraph.swift.
+    - XCUITests should be used, but as a last resort.
+
+### Should your new feature be keyboard navigable?
+
+  * Adding UIKeyCommands will help.
+  * Getting to the new features is as important as using your feature.
+
+### Accessibility for new UI
+
+  * Add accessibilityLabels
+  * Have you tested it with VoiceOver?
```

Comment:
I have never done this, and it is a good idea. Maybe there should be a tiny bit more info about what to do here without too much hand-holding.